examples/lock-app/linux/src/LockManager.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "LockManager.h"
+
+#include <cstring>
+#include <iostream>
+#include <lib/support/logging/CHIPLogging.h>
+
+LockManager LockManager::instance;
+
+LockManager & LockManager::Instance()
+{
+    return instance;
+}
+
+void LockManager::Init()
+{
+    mLocked = false;
```

Comment:
set mLocked in the constructor (i.e. in the header).

Do we even need an Init at this point?

Reply:
Ok. Init removed.